eth/handler.go

```diff
@@ -36,10 +36,8 @@ import (
 	"github.com/ethereum/go-ethereum/rlp"
 )
 
-// This is the target maximum size of returned blocks for the
-// getBlocks message. The reply message may exceed it
-// if a single block is larger than the limit.
-const maxBlockRespSize = 2 * 1024 * 1024
+// This is the target maximum size of returned blocks, headers or node data.
+const softResponseLimit = 2 * 1024 * 1024
```

Comment:
It is only used to limit outbound messages. And it's only a soft limit (i.e. one block will always be sent, irrelevant of it's size). These will probably disappear if/when a chunked network protocol will be introduced at the low level networking layer.